zulip_bots/zulip_bots/bots/ping/doc.md

```diff
@@ -0,0 +1,29 @@
+# Ping Bot
+
+This bot allows users to check availability of a given website or IP
+address across 10 random locations using the free API provided by
+[check-host.net](https://check-host.net). The API documentation can be
+found [here](https://check-host.net/about/api).
+
+### Example Usages
+
+![ping ok](assets/ping-ok.png)
+
+![ping timeout](assets/ping-timeout.png)
+
+### Further Development
```

Comment:
Use ## instead of ### for secondary headers. (Also for the previous paragraphs.)

Reply:
This section "Further Development" is interesting for devs, but probably not so much for users. Maybe open an issue with its content?